src/plugin/cli/src/commonMain/kotlin/community/flock/wirespec/plugin/cli/CommandLineEntitiesParser.kt

```diff
@@ -29,9 +34,9 @@ enum class Options(vararg val flags: String) {
     OutputDir("-o", "--output-dir"),
     Language("-l", "--language"),
     PackageName("-p", "--package"),
+    LogLevel("-ll", "--log-level"),
```

Comment:
It is common practice in clis to use single`-` parameters with only a single letter, like `-d` or `-f`. Multiple character options (or aliases) are prefixed with double dash, e.g. `--language` or `--input-dir`